app/src/main/java/com/hedvig/app/feature/embark/passages/externalinsurer/GetInsuranceProvidersUseCase.kt

```diff
@@ -17,8 +22,22 @@ data class InsuranceProvider(
     val name: String
 )
 
-class GetInsuranceProvidersUseCaseImpl : GetInsuranceProvidersUseCase {
+class GetInsuranceProvidersUseCaseImpl(
+    private val apolloClient: ApolloClient,
+    private val localeManager: LocaleManager
+) : GetInsuranceProvidersUseCase {
```

Comment:
Do we need this Interface-Implementation Pair, or can we inline the interface into the parent class?